runtime-modules/forum/src/lib.rs

```diff
@@ -72,20 +73,12 @@ pub trait ForumUserRegistry<AccountId> {
     fn get_forum_user(id: &AccountId) -> Option<ForumUser<AccountId>>;
 }
 
-/// Convenient composite time stamp
-#[cfg_attr(feature = "std", derive(Serialize, Deserialize, Debug))]
-#[derive(Encode, Decode, Default, Clone, PartialEq, Eq)]
-pub struct BlockchainTimestamp<BlockNumber, Moment> {
-    block: BlockNumber,
-    time: Moment,
-}
-
```

Comment:
@Lezek123 If you can update https://github.com/Joystream/joystream/pull/640 to include this refactoring.
We are dropping `BlockchainTimestamp` type and keeping only `BlockAndTime` struct which are identical.